frontend/src/stores/home.js

```diff
@@ -0,0 +1,40 @@
+import { defineStore } from 'pinia';
+import { get_request } from './services/request_http';
+
+export const useHomeStore = defineStore('homeStore', {
+  state: () => ({
+    home: null,
+    banners: [],
+    home_categories: [],
+    header_categories: [],
+    secondary_categories: [],
+    dataLoaded: false,
+  }),
+  actions: {
+    /**
+     * Fetches the home data from the backend.
+     * Calls the API endpoint to get the data and updates the store's state.
+     * Checks if the data has already been loaded to prevent redundant requests.
+     */
+    async fetchHomeData() {
```

Comment:
Change `fetchHomeData` to `fetchHome` or change `fetchProducts` to `fetchProductsData` to be consistent

Reply:
Changed